tests/integ/test_udf.py

```diff
@@ -1515,6 +1557,36 @@ def test_add_requirements(session, resources_path):
     assert session.get_packages() == {
         "numpy": "numpy==1.23.5",
         "pandas": "pandas==1.5.3",
+        "snowflake-snowpark-python": "snowflake-snowpark-python",
```

Comment:
New behavior - Snowpark is added by default.

Reply:
Just to confirm: this will not a breaking change since we always allow `snowflake-snowpark-python` to be overwritten?

Reply:
Yeah, the version can always be overwritten. I will add a specific test case to show that this is true.

Reply:
test_add_requirements_and_override_snowpark_package added